swag/blocks/__init__.py

```diff
@@ -0,0 +1,3 @@
+from .cagnotte_blocks import *
+from .swag_blocks import *
+from .system_blocks import *
```

Comment:
Hum, yes, it could, but no, we can't. Since $wagChain is already in production, we are tied to the current public API and the names of existing blocks are part of that API; breaking that API means we need to rebuild the entire blockchain and write another converter and all. But we can discuss that in prevision of SwagChain v2.